### ISSUE TITLE
Bug 54110468: [React Native Gallery -> VirtualizedList]: Selected list item is not visible to the user in high contrast aquatic theme.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#500]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: High Contrast 
 

Pre-Requisite:  

Turn on High Contrast Aquatic Theme (Settings->Accessibility->Contrast themes->Aquatic/Dessert theme->Apply)

Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘VirtualizedList’ button present under “Collection” group and invoke it.
Navigate using tab key to any item present under "A VirtualizedList with single selection support, heading" and select it.
Now observe the selected list item in high contrast themes.
Actual Result:
Selected list item is not visible to the user in high contrast aquatic theme.

Note: Same issue is observed in high contrast desert theme also.

Expected Result:
Selected list item also should be visible clearly to the user in both high contrast aquatic themes.

User impact:

Users with low vision will not be able to know about the selected list item if it is not visible in High Contrast Themes.

Link to ADO bug: https://dev.azure.com/microsoft/OS/_workitems/edit/54110468
